Python/Product/Cookiecutter/ViewModel/CookiecutterViewModel.cs

```diff
@@ -476,16 +476,20 @@ class CookiecutterViewModel : INotifyPropertyChanged {
             SearchResults.Add(Recommended);
             SearchResults.Add(GitHub);
 
+            // Ensure that there's a selection, to avoid focus issues
+            // when tabbing to the search results.
+            if (SearchResults.Where(cat => cat.IsSelected).Count() == 0) {
```

Comment:
Prefer `SearchResults.Any(cat => cat.IsSelected)` to save enumerating every single item.